tools/free_energy/.shed.yml

```diff
@@ -0,0 +1,12 @@
+name: freeenergy
+owner: chemteam
+description: "Free energy tools of BRIDGE."
+remote_repository_url: https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/free_energy
+type: unrestricted
+categories:
+- Free Energy
```

Comment:
those categories needs to exist in the ToolShed ... free energy does not afaik

Reply:
@bgruening should I remoeve that?

Reply:
yes please

Reply:
@bgruening Done!!